### PR TITLE
Addresses #524: Susper responsive on mobile small screen size 320px

### PR DESCRIPTION
**Pull request by harshit98**
Addresses issue #524 

Changes: Made Susper responsive on smaller mobile screen size (320px)

Demo Link: https://harshit98.github.io/susper.com/

Screenshots for the change: 

(Not applicable here, since there will be too many screenshots)

@mariobehling @nikhilrayaprolu @Marauderer97 Please review carefully checking all the features. Let me know if some changes have to be done or not.

### CODE REVIEW

File: src/app/index/index.component.css [hunk 1]
```diff
@@ -171,9 +171,9 @@ footer {
   }
 }
 
-@media screen and (max-width: 479px) {
```
Comment (Marauderer97) [inline src/app/index/index.component.css]: @harshit98 Please be extra careful when you change css, this was very painstakingly tested and written to ensure mobile responsiveness. Please when you make changes here
Before your change: 
image

After your change:
image

**Comment by harshit98** [comment]
@Marauderer97 Please check only for 320px. I'm following chrome's responsiveness feature which is quite good.

File: src/app/results/results.component.css [hunk 2]
```diff
@@ -613,26 +607,17 @@ div.autocorrect{
...
-  #search-options {
-    margin-left: 7%;
-  }
   #search-options li {
     padding: 0px 4vw 12px;
   }
 }
 
 @media screen and (max-width: 460px) {
-  .feed.col-md-10.col-md-offset-1.col-sm-10.col-sm-offset-1.col-lg-10.col-lg-offset-1 {
```
Comment (Marauderer97) [inline src/app/results/results.component.css]: Like @nikhilrayaprolu pointed out this is a bad class name, I fixed it in the large screen pull, please wait for it to get merged, then we will have lesser problems.

File: src/app/results/results.component.css [hunk 3]
```diff
@@ -654,3 +639,23 @@ div.autocorrect{
     padding: 0px 2vw 12px;
   }
 }
+
+@media screen and (max-width: 320px) {
+  #search-options {
+    margin-left: 0.3%;
+  }
+  .feed.col-md-10.col-md-offset-1.col-sm-10.col-sm-offset-1.col-lg-10.col-lg-offset-1 {
```
Comment (Marauderer97) [inline src/app/results/results.component.css]: There is no need for this, the large screen pull automatically fixes this, please let it get merged

**Comment by harshit98** [comment]
@Marauderer97 Okay, I working on it. :+1:

**Comment by harshit98** [comment]
I'm closing this PR since there are some new changes done in #688

**Pull request closed without merging**